### PR TITLE
Merge in release 6.x into master

### PR DESCRIPTION
**Pull request by andrey-skl**

### CODE REVIEW

**Review by skoch13 — APPROVED** [review]

File: packages/screenshots/testplane/test.mjs [hunk 1]
```diff
@@ -5,7 +5,6 @@ import filenamify from 'filenamify';
 
 import Actions from './actions.js';
 
-// eslint-disable-next-line import/no-unresolved
```
Comment (Hypnosphi) [inline packages/screenshots/testplane/test.mjs:8]: this is a generated file that's absent until you run visual tests

Comment (andrey-skl) [reply packages/screenshots/testplane/test.mjs:8]: So I should delete it? Sure, thanks!

Comment (Hypnosphi) [reply packages/screenshots/testplane/test.mjs:8]: Yes, and it means that we still need a eslint-disable here

Comment (andrey-skl) [reply packages/screenshots/testplane/test.mjs:8]: Fun part is that one have to hack this around : D

**Review by Hypnosphi — APPROVED** [review]

**Pull request merged**